recipes/update_head_node_slurm.rb

```diff
@@ -35,9 +35,10 @@
   end
   # Generate pcluster specific configs
   execute "generate_pcluster_slurm_configs" do
+    no_gpu = nvidia_installed?
```

Comment:
this needs to be something like 
```
no_gpu = nvidia_installed? ? "--no-gpu" else ""
```

and then only use `#{no_gpu}` in the command. The same needs to be done in head_node_slurm_config

Reply:
Changed